tap_tiktok_ads/__init__.py

```diff
@@ -27,10 +27,11 @@ def main():
         raise Exception("Provided list of account IDs contains invalid IDs. Kindly check your Account IDs.") from None
     # update string to list in the config
     args.config['accounts'] = accounts_list
+    sandbox = True if args.config.get('sandbox', False) else False
```

Comment:
In above conditional statement `{"sandbox": "false"}' will also return the 'True' value. We may need to evalute falsy string after reading it from the config or do a string comparison.

Reply:
Also, tt looks like we are assuming sandbox value to be bool but `README.md` states otherwise. If not this, we at least need to update the README.md.